src/components/form-slice/FormPartSSL/FormItemCertKeyList.tsx

```diff
@@ -0,0 +1,107 @@
+import { Button, Fieldset, Stack, type FieldsetProps } from '@mantine/core';
+import { useTranslation } from 'react-i18next';
+import { useFieldArray, useFormContext } from 'react-hook-form';
+import { FormSection } from '../FormSection';
+import type { SSLPostType } from './schema';
+import type { PropsWithChildren } from 'react';
+import { FormItemTextareaWithUpload } from '@/components/form/TextareaWithUpload';
+import IconDelete from '~icons/material-symbols/delete-forever-outline';
+
+const PairWrapper = (
+  props: PropsWithChildren & Pick<FieldsetProps, 'legend'>
+) => {
+  const { children, legend } = props;
+  return (
+    <Fieldset p={8} mb={5} legend={legend}>
+      <Stack flex={1} gap={1}>
+        {children}
+      </Stack>
+    </Fieldset>
+  );
+};
+
+const RequiredCertKey = () => {
+  const { t } = useTranslation();
+  const { control } = useFormContext<SSLPostType>();
+  return (
+    <PairWrapper>
+      <FormItemTextareaWithUpload
+        control={control}
+        label={`${t('form.ssls.cert')} 1`}
```

Comment:
static number?

Reply:
yes, it's a feature. This makes the required items cert and key as the first items before certs and keys.

Reply:
I will add comments in code. tracked in #3000